dataprofiler/profilers/profiler_options.py

```diff
@@ -851,6 +852,10 @@ def __init__(self, is_enabled=True, is_case_sensitive=True,
         :vartype is_case_sensitive: bool
         :ivar stop_words: option set for stop words.
         :vartype stop_words: Union[None, list(str)]
+        :ivar top_k_chars: option set for top common characters.
```

Comment:
Can this be explained more?

Reply:
added a bit more detail. Does it look better for you?